qiskit_nature/second_q/operators/polynomial_tensor.py

```diff
@@ -71,6 +87,15 @@ def register_length(self) -> int:
         """Returns register length of the operator key in `PolynomialTensor`."""
         return self._register_length
 
+    @classmethod
+    def empty(cls, register_length: int | None) -> PolynomialTensor:
+        """TODO."""
+        return PolynomialTensor({}, register_length=register_length)
+
+    def is_empty(self) -> bool:
+        """TODO."""
+        return self == PolynomialTensor.empty(self.register_length)
```

Comment:
In similar vain to #831 we could also allow `register_length=None` and determine it lazily once the internal data has contents